### PR TITLE
Openshift VCR-recording script, touch up minishift instructions.

### PR DESCRIPTION
**Pull request by cben**
Followups from https://github.com/ManageIQ/manageiq-providers-openshift/pull/75

### CODE REVIEW

**Comment by cben** [comment]
@enoodle @moolitayer please review

**Comment by cben** [comment]
@zeari please review

File: providers/openshift.md [hunk 1]
```diff
@@ -83,49 +83,43 @@ will assume that is being used, and updates to some commands might be
...
+  $ git clone https://gist.github.com/e2fac8be87ea0e9f429b6f5d75e02176 ~/minishift/addons/manageiq
+  $ minishift addons install --force ~/minishift/addons/manageiq
   $ minishift addons enable manageiq
   ```
   
-3. Start minishift:
+2. Start minishift:
   
   ```console
   $ minishift start --vm-driver virtualbox --openshift-version "v1.5.0-rc.0"
```
Comment (zeari) [inline providers/openshift.md]: Is this the correct openshift version?

Comment (cben) [reply providers/openshift.md]: updated to v3.6.1 which is what I used to record currently.
doesn't matter much, there is a link couple lines below to all valid tags.

**Comment by cben** [comment]
@agrare @blomquisg please review/merge (not sure who are maintainers in this repo)

**Comment by cben** [comment]
Updated per https://github.com/ManageIQ/manageiq-providers-openshift/pull/83.

@chessbyte @Fryguy Who can merge PRs in this repo?

**Pull request merged**